andela/mmdp_site
# MMDP-771 Fix LGA information to only capture specified LGA

**What does this PR do?**

fixes stakeholder endpoint returning beneficiaries located in communities not within the specified LGA

**description of Task to be completed?**

- filter beneficiaries by LGA

**How should this be manually tested?**

- `git fetch`
- `git checkout bg-fix-lga-information-to-only-capture-specified-LGA-MMDP-771 `
- `yarn dev`
-  use postman to hit `http://localhost:3000/api/v1/location?lga=Akoko-Edo&focusAreaName` 
- notice that for each stakeholder, under the beneficiaries dictionary, communities returned are now only from the said LGA

**Any background context you want to provide?**

This bug fix affects our LGA level coordination matrix page, ensure data returned by endpoint does not include communities not within specified LGA

**What are the relevant Jira issues?**

[MMDP-771](https://viisaus.atlassian.net/browse/MMDP-771)

**Screenshots (if appropriate)**

N/A
